addons/bus/static/src/js/crosstab_bus.js

```diff
@@ -342,7 +342,7 @@ var CrossTabBus = Longpolling.extend({
      */
     _onUnload: function () {
         // unload peer
-        var peers = this._callLocalStorage('getItem', 'peers', {});
+        var peers = this._callLocalStorage('getItem', 'peers') || {};
         delete peers[this._id];
```

Comment:
@VincentSchippefilt after improving logging handling so we get proper error reporting of loading issues on runbot, it reports an issue here with "cannot convert undefined or null to object", I tried finding out why but got ~~high~~ lost amongst the indirection.

Even with the logging improvements of this PR, the debugging experience is not exactly stellar, and this issue doesn't seem to occur when running in Chrome/Chromium (and obviously nothing works in firefox). So this fixes the symptoms but not the underlying issue that there are apparently situations where the default-setting simply doesn't work. I'd guess an issue with the callback being called asynchronously in https://github.com/odoo/odoo/blob/236b8f120bb23aa569397cf4dc6b881d6f41155a/addons/web/static/src/js/core/service_mixins.js#L113-L125

Reply:
@xmo-odoo @alexkuhn I'm forwarding to AKU, for crosstab bussing things ;-)

Reply:
The correct fix has been applied in 13.1, could be backported in 13.0:
https://github.com/odoo/odoo/commit/597f12a7578f14b483260ddb941ff240feccfd35

Reply:
nevermind... it has been fixed in 12.0 and above :)